clients/fossid-webapp/src/test/kotlin/FossId2025dot1SnippetMappingTest.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2025 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.clients.fossid
+
+import io.kotest.core.spec.style.StringSpec
+
+import org.ossreviewtoolkit.clients.fossid.model.result.Snippet
+
+class FossId2025dot1SnippetMappingTest : StringSpec({
```

Comment:
A file / class naming nit: While currently the only tested thin indeed is the snippet mapping, just "FossId2025dot1Test" would better align with existing tests and pave the way for easily adding other version-2025-related tests here.